tagsets/orsee_mysql.php

```diff
@@ -247,5 +247,121 @@ function dump_array($array,$title="",$dolang=true) {
     echo '</TABLE>';
 }
 
+function check_database_upgrade() {
+    global $settings, $system__database_version;
+
+    if (!isset($settings['database_version'])) {
+        $settings['database_version']='2017112700';
+        $done=or_query("INSERT INTO ".table('options')." VALUES (1,'general',NULL,'database_version','".$settings['database_version']."')");
+    }
+    if ($settings['database_version']<$system__database_version) {
```

Comment:
OK, this one is important. Please make sure you cast each value used in comparison to int (otherwise you compare strings).

Reply:
Which was one reason why I defined the version number as a it as a 10-character string. But ok, I now cast all uses of database_version as (int).